app/components/TaskView.vue

```diff
@@ -311,6 +369,36 @@ const openTaskInNewTab = () => {
   window.open(`/channel/${props.cid}/task/${props.taskId}`, '_blank')
 }
 
+const onFileUpload = async (files) => {
+  if (!files.length) return
+
+  for (const file of files) {
```

Comment:
async operations in loops for no good. Just push them into resolver array and use `Promise.allSettled`. Make sure to use error toast if anyone of them are failed to upload, show them as error text message.